version: "3.9"

services:
  db:
    image: postgres
    volumes:
      - ./data/db:/var/lib/postgresql/data
    environment:
      - POSTGRES_DB=postgres
      - POSTGRES_USER=postgres
      - POSTGRES_PASSWORD=postgres
  web:
    build: .
    command: python manage.py runserver 0.0.0.0:8000
    volumes:
      - .:/code
    ports:
      - "8000:8000"
    environment:
      - USE_RDS_DB=True
      - RDS_DB_NAME=postgres
      - RDS_USERNAME=postgres
      - RDS_PASSWORD=postgres
      - RDS_HOSTNAME=db
      - RDS_PORT=5432
      - DEBUG=True
      - SECRET_KEY=%^2xz0q43$i2+fg=w9o)@+mvhw*4j6+!uu-1q@eh!g1yj8c&_d
      - FRONT_URL=http://127.0.0.1:8080
      - USE_S3=False
      - BASE_URI=https://auth.dpvonline.de/
      - OIDC_RP_CLIENT_ID=anmelde-tool
      - OIDC_RP_REALMNAME=DPV
      - OIDC_RP_CLIENT_SECRET_BACKEND=p9lHwixCFiRp6Cr187uxNtSFC2KhZppD
      - USE_SES=False
    depends_on:
      - db


